src/main/java/org/sagebionetworks/web/client/security/AuthenticationControllerImpl.java

```diff
@@ -61,8 +61,8 @@ public class AuthenticationControllerImpl implements AuthenticationController {
     NIH_NOTIFICATION_DISMISSED,
     COOKIES_ACCEPTED,
   };
-  private static String currentUserAccessToken;
-  private static UserProfile currentUserProfile;
```

Comment:
Am I correct in assuming that we won't have issues removing `static` because this is a singleton class? We were never accessing these in a static context, so no additional changes were required by this change.